components/footer/_footer.scss

```diff
@@ -34,3 +34,18 @@
     }
   }
 }
+.footer--eq {
+    .footer__link {
+      float: left;
+      margin-right: 1.5rem;
+    }
+  .footer__license {
+      margin-top: 1.5rem;
+      border-top: 1px solid $color-borders;
+      padding-top: 1.5rem;
+      .footer__link {
```

Comment:
This should be a modifier or a utility class on the `<li>` 
Either `class="footer__list--inline"` or `class="u-inline"`